src/test/datascience/ipywidgets/ipyWidgetScriptManager.vscode.common.test.ts

```diff
@@ -66,6 +70,23 @@ suite('IPyWidget Script Manager @widgets', function () {
             disposables
         );
         const { notebook, editor } = await openNotebook(testWidgetNb);
+        if (!isWeb()) {
+            // Create the controller and select it for the tests.
```

Comment:
Now that we're lazy loading controllers, we should create the controller for the tests,
this is much faster and more reliable, will also be required when we adopt the new kernel picker (as user will always have to select a kernel)

Reply:
This fixes a flaky test with Widgets